mainnet-contracts/src/WithdrawalManager.sol

```diff
@@ -95,23 +79,26 @@ contract WithdrawalManager {
     }
 
     /**
-     * @dev Finalize withdrawals for a batch or multiple batches
+     * @notice Finalizes the withdrawals up to the given batch index
      * @param withdrawalBatchIndex The index of the last batch to finalize
+     * @dev Restricted to the Guardian
```

Comment:
Should we leave it like this, or use a different convention?
For the FWR I've used a comment like 
`     * @dev Restricted access to `ROLE_ID_OPERATIONS_PAYMASTER` wyt?

Reply:
Yes, the guardian is responsible for double checking the provisioning. The SGX enclave handles that logic. The tx broadcaster is this role `ROLE_ID_OPERATIONS_PAYMASTER`.
We maya asign different roles to different hot wallets